tyk-oss/README.md

```diff
@@ -103,8 +103,9 @@ To enable Pump, set `global.components.pump` to true, and configure below inside
 |---------------------------|------------------------------------------------------------------------------------------------------------| 
 | Prometheus Pump (Default) | Add `prometheus` to `pump.backend`, and add connection details for prometheus under `pump.prometheusPump`. |
 | Mongo Pump                | Add `mongo` to `pump.backend`, and add connection details for mongo under `.global.mongo`.                 |
-| SQL Pump                  | Add `postgres` to `pump.backend`, and add connection details for postgres under `.global.postgres`.        |
+| SQL Pump                  | Add `postgres` to `.pump.backend`, and add connection details for postgres under `.global.postgres`.       |
```

Comment:
Shouldn't we add `postgres` to `tyk-pump.pump.backend` instead of `.pump.backend`?

Reply:
I'm trying to make the configuration section more reusable so that I can copy from component chart README directly to the umbrella chart README.

For the umbrella chart, I've added this at the beginning of the Configuration section:
"To enable Pump, set global.components.pump to true, and configure below inside tyk-pump section."

What do you think about this structure? Is there any better way to update this?

Reply:
We can try moving field names such as `.pump.backend` to the variables. After directly copying READMEs, changing environment variables should be sufficient. So, the only required change will be environment variables.

Something like the following:

```markdown
<!---
Declare variables at the beginning of the doc
-->
[pump-backend]: ./tyk-pump.pump.backend 

<!---
... rest of the docs
-->


<!---
instead of `.pump.backend`, use [pump-backend], 
which can be configurable from the declaration above for each readme. 
-->
| SQL Pump | Add `postgres` to `[pump-backend]` ... |

```

Reply:
I'll open a ticket 💯